Description: fix mysql log and sock paths for Debian
Author: Valentin Vidic <Valentin.Vidic@CARNet.hr>
Last-Update: 2017-12-03
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/heartbeat/mysql-common.sh
+++ b/heartbeat/mysql-common.sh
@@ -30,15 +30,15 @@
 	OCF_RESKEY_socket_default="/var/run/mysql/mysql.sock"
 else
 	if [ "$OCF_RESKEY_binary_default" = "" ]; then
-		OCF_RESKEY_binary_default="/usr/bin/safe_mysqld"
+		OCF_RESKEY_binary_default="/usr/bin/mysqld_safe"
 	fi
-	OCF_RESKEY_config_default="/etc/my.cnf"
+	OCF_RESKEY_config_default="/etc/mysql/my.cnf"
 	OCF_RESKEY_datadir_default="/var/lib/mysql"
 	OCF_RESKEY_user_default="mysql"
 	OCF_RESKEY_group_default="mysql"
-	OCF_RESKEY_log_default="/var/log/mysqld.log"
-	OCF_RESKEY_pid_default="/var/run/mysql/mysqld.pid"
-	OCF_RESKEY_socket_default="/var/lib/mysql/mysql.sock"
+	OCF_RESKEY_log_default="/var/log/mysql/error.log"
+	OCF_RESKEY_pid_default="/var/run/mysqld/mysqld.pid"
+	OCF_RESKEY_socket_default="/var/run/mysqld/mysqld.sock"
 fi
 OCF_RESKEY_client_binary_default="mysql"
 OCF_RESKEY_test_user_default="root"
